doc/website/src/pages/knowledge.md

```diff
@@ -0,0 +1,16 @@
+# Knowledge Center
+
+<br></br>
+
+Welcome to the Atlas Knowledge Center.
+
+<br></br>
+
+## Databases
+
+<br></br>
```

Comment:
This is a placeholder page for the knowledge center homepage and is likely to be changed. Regardless, I added newlines because otherwise it looks too compact.